server/views/upload-document.html

```diff
@@ -35,6 +37,7 @@
           </span>
         {% endif %}
           
+        <div id="notUploading1" style="display:block">
```

Comment:
Avoid inline styles (there are multiple occurrences of this), use GOV UK classes instead:

https://design-system.service.gov.uk/styles/layout/#override-how-elements-display